test/test_circuit_builder.py

```diff
@@ -19,12 +19,24 @@ def test_simple(self):
 
         assert circuit.qubit_register_size == 2
         assert circuit.qubit_register_name == "q"
-        assert circuit.squirrel_ir.statements == [
+        assert circuit.ir.statements == [
             Comment("A single line comment."),
             H(Qubit(0)),
             CNOT(Qubit(0), Qubit(1)),
         ]
 
+    def test_identity(self):
```

Comment:
We should also add one to the integration, i.e test to see if the identity gate can be read on cqasm and incorporated on the IR

Reply:
I have updated `test_libqasm.py`, `test_simple`, adding a `I q[0]` line. That is basically doing the cQASM v3 parsing and creating an IR, then comparing it with another circuit built through the circuit builder.

Let me know if you think that's enough. I can add a specific integration test otherwise, or update an existing integration test.